src/components/index.js

```diff
@@ -0,0 +1,5 @@
+import { Header } from "./header/Header";
```

Comment:
suggestion: use prettier for code formatting

Reply:
Done!